p2p/__init__.py

```diff
@@ -0,0 +1,2 @@
+# This is to ensure we call setup_trace_logging() before anything else.
+import evm  # noqa: F401
```

Comment:
Just pointing out that this won't work as a long term solution as when `p2p` is extracted from his codebase it won't have access to the `evm` module.  Maybe we should package up our trace logging facilities in a small re-usable package.

Reply:
Maybe `eth_utils`?